src/pages/siswa/academic/nilai/nilai_siswa.jsx

```
@@ -113,261 +70,165 @@ const dataKeterampilan = [
     </div>
-  );
-};
+  )
+}
+
+const NilaiSiswa = () => {
+  const kelasTersedia = useMemo(() => {
+    return [...new Set([...dataPengetahuan, ...dataKeterampilan].map((d) => d.kelas))]
+  }, [])
+
+  const [kelasAktif, setKelasAktif] = useState(kelasTersedia[0])
```

ESLint

[error] 'setKelasAktif' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)